src/config/comfySettings.ts

```diff
@@ -39,11 +41,16 @@ export class ComfySettings {
     this.settings = JSON.parse(fileContent);
   }
 
+  public saveSettings() {
+    fs.writeFileSync(this.filePath, JSON.stringify(this.settings), 'utf-8');
```

Comment:
This is very problematic as the file content can be newer than the snapshot state when we read it during electron app start.